app/components/support_interface/application_summary_component.rb

```diff
@@ -140,6 +148,10 @@ def formatted_status
       "<strong>#{name}</strong><br>#{desc}".html_safe
     end
 
+    def one_login?
+      application_form.candidate.one_login_auth.present?
```

Comment:
wouldn't it be lovely to have something like 
```suggestion
      candidate.one_login_connected?
```
Very minor suggestion and not a blocker 👌

Reply:
Yes definitely. I'm sure this will come in handy. Thank you 🙏🏼 

https://github.com/DFE-Digital/apply-for-teacher-training/pull/10295/commits/be6750ead6b69046effddaf1f5c87ae942f7bb8c